### PR TITLE
added nav bar, changed some styling

### PR DESCRIPTION
**Pull request by katch00**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -11,15 +11,43 @@
...
   	margin-left: auto;
 	margin-right: auto;
 }
+
+ul { 
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
+	background-color: #51667b;
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/style.css:36]: Style Nit: Alphabetize declarations.

From style guide:
Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.

Ignore vendor-specific prefixes for sorting purposes. However, multiple vendor-specific prefixes for a certain CSS property should be kept sorted (e.g. -moz prefix comes before -webkit).

ex:
background: fuchsia;
border: 1px solid;
-moz-border-radius: 4px;
-webkit-border-radius: 4px;
border-radius: 4px;
color: black;
text-align: center;
text-indent: 2em;

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -11,15 +11,43 @@
...
+}
+
+li {
+	float: left;
+}
+
+li a { 
+	display: block;
+	color: white;
+	text-align: center;
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/style.css:46]: Same style nit

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -11,15 +11,43 @@
 h1 {
 	color: white;
 	text-align: center;
+	font-family:"Courier New", Courier, monospace;
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/style.css:14]: Same style nit

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -5,7 +5,14 @@
 }
 
 #greeting-container {
-  margin-top: 20px;
+        margin-top: 20px;
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/style.css]: Extra indentation

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -5,7 +5,14 @@
...
+	font-family:"Courier New", Courier, monospace;
+	font-size: 30px;
+	color: white;
+	margin-left: auto;
+	margin-right: auto;
+	width: 80vw;
+	text-align: center;
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/style.css]: Nit, and I realize this wasn't being followed in the starter code, but css declarations should be in alphabetical order. Don't feel the need to fix every occurrence now though

From the style guide:
Alphabetize declarations.

Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.

Ignore vendor-specific prefixes for sorting purposes. However, multiple vendor-specific prefixes for a certain CSS property should be kept sorted (e.g. -moz prefix comes before -webkit).
ex:
background: fuchsia;
border: 1px solid;
-moz-border-radius: 4px;
-webkit-border-radius: 4px;
border-radius: 4px;
color: black;
text-align: center;
text-indent: 2em;

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -46,8 +53,47 @@ li a {
...
+	display: inline-block;
+	font-size: 24px;
+	margin: 4px 2px;
+	transition-duration: 0.4s;
+	cursor: pointer;
+	border-radius: 14px;
+	color: white;
+	font-family: "Courier New", Courier, monospace;
+	margin: auto;
+        display: flex;
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/style.css]: Extra indenting

File: portfolio/src/main/webapp/images/projImages/info [hunk 7]
```diff
@@ -0,0 +1 @@
+This folder contains images to be used in the project page
```
Comment (shawntabai) [inline portfolio/src/main/webapp/images/projImages/info:1]: You may want to consider using all lowercase for file/folder names. Case sensitivity in filenames is not consistent across different OSes and server frameworks, and it's an easy way to spend hours trying to figure out why a file won't load (in my experience at least).

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -1,25 +1,108 @@
 #content {
 	margin-left: auto;
 	margin-right: auto;
-	width: 650px;
+	width: 80vw;
 }
 
 #greeting-container {
-  margin-top: 20px;
+	color: white;
```
Comment (shawntabai) [inline portfolio/src/main/webapp/style.css:8]: Might be good to fix inconsistent indentation here. I'm guessing some lines were copied in with tab indents, and some lines were hand-written and the editor used space indents (happens all the time). Your editor may have an option to normalize indentation.

**Pull request merged**